eclass/kernel-build.eclass

```diff
@@ -547,6 +556,17 @@ kernel-build_merge_configs() {
 		fi
 	fi
 
+	# Only semi-related but let's use that to avoid changing stable ebuilds.
+	if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
```

Comment:
Why do we need to make this conditional to `${KERNEL_IUSE_GENERIC_UKI} ]]`? If we put it in a new version of gentoo-kernel-config, then it also doesn't effect the kernel versions we have in the tree already right?

Reply:
Hmm, that's another option. However, then we wouldn't have a clean way or orchestrating the USE flag from eclass.